config/configmap/inferenceservice.yaml

```diff
@@ -63,7 +63,9 @@ data:
            "memoryLimit": "1Gi",
            "cpuRequest": "100m",
            "cpuLimit": "1",
-           "enableDirectPvcVolumeMount": false
+           "caBundleSecretName": "",
+           "caBundleVolumeMountPath": "/etc/ssl/custom-certs",
```

Comment:
Given most of these fields may go away, I'm no longer sure this is the right place for new stuff. Maybe new stuff should go only to the ClusterStorageContainer (see [doc](https://docs.google.com/document/d/1UcHFCQcTcDBcZVd8MTEgSXXKr7GMpzHpjuJW8gEwyFU/edit)). I'll let @yuzisun to comment about this.

In any case, minimally the mount path should be replicated in the default ClusterStorageContainer and whatever is in the CRD should have priority over the global config, since we don't know what's going to be the right place for any given container.

Reply:
yes, there will be a new place to specify these fields but we don't know when the new CRD will come, so for now, this configmap is the right place to use.

Reply:
The CRD is already there: https://github.com/kserve/kserve/blob/master/config/crd/serving.kserve.io_clusterstoragecontainers.yaml

Reply:
is it already implemented?

Reply:
I think yes. I've never tried it. But here is the PR that introduced it: https://github.com/kserve/kserve/pull/3060

Reply:
@greenmoon55 do you think I should create these fields in the new CRD(clusterstoragecontainers)?

Reply:
Discussed this in the community meeting and we agreed that we don't need to add this to `clusterstoragecontainers`